src/test/java/com/iexec/core/replicate/ReplicateSupplyServiceTests.java

```diff
@@ -106,7 +106,7 @@ void shouldNotGetAnyReplicateSinceWorkerDoesNotExist() {
         Optional<WorkerpoolAuthorization> oAuthorization =
                 replicateSupplyService.getAuthOfAvailableReplicate(workerLastBlock, WALLET_WORKER_1);
         assertThat(oAuthorization).isEmpty();
-        Mockito.verifyNoInteractions(web3jService, taskService, contributionTimeoutTaskDetector, taskUpdateManager, replicatesService, signatureService, smsService);
+        Mockito.verifyNoInteractions(web3jService, taskService, contributionTimeoutTaskDetector, replicatesService, signatureService, smsService);
```

Comment:
```suggestion
        Mockito.verifyNoInteractions(web3jService, taskUpdateRequestManager, taskService, contributionTimeoutTaskDetector, replicatesService, signatureService, smsService);
```
I think `taskUpdateManager` should have been replaced with `taskUpdateRequestManager` in some tests.
This is to ensure we never call some objects which mean we did not go on some code paths.
There is a loss of precision with the last fix imho

Reply:
Right, updated in 8b05a75.